decidim-core/app/forms/decidim/base_form.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+module Decidim
+  # A base form object to hold common logic, like automatically adding as
+  # attributes the params sent as context by the `FormFactory` concern.
+  class BaseForm < Rectify::Form
```

Comment:
I don't get this:

>  Sounds like me like this could be a potential security vulnerability, in case you send the attributes from the view (they would be overriden!)

A form per se does not perform anything against the database. It just checks that the values sent by the form are valid, that's all. I don't want to put them as `private` as it misses the whole point of using a custom validator that needs access to the current organization.